graphene_django/tests/test_types.py

```diff
@@ -174,6 +174,7 @@ def test_schema_representation():
 enum ReporterAChoice {
   A_1
   A_2
+  A_3
```

Comment:
I can't see the corresponding code change for this schema update. Why has this been added?

Reply:
A third option was added in the test models.py, it starts with an underscore and has a nonascii code.

Reply:
👍